src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/ChoiceOrikaConverter.java

```diff
@@ -105,6 +109,8 @@ public Choice convertFrom(final ChoiceDTO source, final Type<Choice> destination
             return super.mapperFacade.map(source, GraphChoice.class);
         } else if (source instanceof StringChoiceDTO) {
             return super.mapperFacade.map(source, StringChoice.class);
+        } else if (source instanceof RegexPatternDTO) {
+            return super.mapperFacade.map(source, StringChoice.class);
```

Comment:
See my comment above, but this I think must be `RegexPattern` too, for the same reason.

Reply:
Fixed in most recent commit